syncr_backend/external_interface/dht_util.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
+from typing import Any
+from typing import List
+from typing import Tuple
+
+from kademlia.network import Server  # type: ignore
+from kademlia.storage import ForgetfulStorage  # type: ignore
+
+from syncr_backend.util import crypto_util
+from syncr_backend.util.log_util import get_logger
+
+
+logger = get_logger(__name__)
+
+_node_instance = None
+
+
+def get_dht() -> Server:
+    """
```

Comment:
Can you add to the docstring here that this will throw an exception if the dht hasn't been initialized?

Reply:
Please do this